src/main/java/nl/tudelft/ti2206/group9/gui/ShopScene.java

```diff
@@ -49,7 +53,40 @@
                 .toString(State.getCoins()), 4, 24);
 
         setButtonFunction(backButton, BType.SHOP_BACK);
-        return new Node[]{backButton, coinsLabel, amountLabel, itemList};
+        return new Node[]{backButton, coinsLabel, amountLabel, itemTable};
+    }
+
+    /**
+     * Method to fill the table with skins.
+     */
+    private static void setUpTable() {
+        items.clear();
+        itemTable.getColumns().clear();
+        itemTable.setItems(items);
+        items.addAll(Style.getAndy(), Style.getBoy(),
+                Style.getCaptain(), Style.getIronMan(),
+                Style.getNoob(), Style.getPlank());
```

Comment:
Maybe there should be a `Style.getAllSkins()` method returning a collection? Then you can still use addAll :)

<logic,review